packages/esm-patient-programs-app/src/programs/program-actions-menu.component.test.tsx

```diff
@@ -0,0 +1,70 @@
+import { showModal } from '@openmrs/esm-framework';
+import { ProgramsActionsMenu } from './programs-actions-menu.component';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import React from 'react';
+import { launchPatientWorkspace } from '@openmrs/esm-patient-common-lib';
+
+jest.mock('@openmrs/esm-framework', () => ({
+  showModal: jest.fn(),
+  useLayoutType: jest.fn(() => 'desktop'),
+}));
+
+jest.mock('@openmrs/esm-patient-common-lib', () => ({
+  launchPatientWorkspace: jest.fn(),
+}));
+
+jest.mock('react-i18next', () => ({
+  useTranslation: () => ({ t: (key: string, defaultText: string) => defaultText }),
+}));
```

Comment:
Same as above, we shouldn't mock this. A lot of the tips from the programs modal test can be applied here too.